master_xml.py

```diff
@@ -209,6 +212,7 @@ def createCardJson():
 
         if (template.find('Tooltip') != None):
             card['info'] = {}
+            card['infoFormatted'] = {}
             for region in LOCALES:
                 # Set to tooltipId for now, we will evaluate after we have looked at every card.
                 card['info'][region] = template.find('Tooltip').attrib['key']
```

Comment:
It's just unnecessary logic. The tooltip id only needs to be in one of the fields for us to be able to evaluate it.